release_todos

```diff
@@ -0,0 +1,14 @@
+When ready to do a release, update the following:
+
+1. conf.h - update version number and copyright eg current year
+2. Changelog - list changes
+3. debian/changelog - list changes
+4. debian/copyright - update copyright dates as required eg current year
+5. docs/source/conf.py - update copyright notice eg current year, version and release numbers
+6. "cd docs" then "make man". Copy resulting docs/build/man/nodogsplash.1 to overwrite old debian/doc/nodogsplash.1
+7. remove /docs/build directory created in step 6
+8. Push to origin
+9. Github - Create Pull Request and Merge to master
+Finally on Github proceed to "Draft New Release"
```

Comment:
Shouldn't this be part of the docs? Note files in the root folder will become messy.

Reply:
Also mention the release tagging via `git tag -a v3.3.0 -m "nodogsplash release 3.3.0 "` and `git push --tags`.

Reply:
You have a valid argument but I feel it is important this note is left in plain sight - we both have trouble remembering all the things to do :-D (have I remembered all of them for this note? lmao)

Reply:
Ha! yes indeed - the dreaded release tagging!

Reply:
Or put it into the wiki. Otherwise we open the door for other howto and todo files.

Reply:
The door is open only if we leave it open. I agree for anything else but this really does need to be in plain sight.

Reply:
It is important to me. Please put it in the docs or the wiki.

Reply:
Wherever it goes it is important that everyone doing a release sees it.
Docs is for "How to use", so how about a "Notes" folder? Then all the howto todos can go in there, keeping things tidy.

Reply:
I'd forgotten there even was a wiki...
Looks like I even added something once too. Must be getting old :-D
I wonder if anyone looks at it.

Reply:
OK, It is now in a Notes folder and also in the wiki ;-)

Reply:
Ok, then we need no additional file in the root folder then.

Reply:
Agreed ;-)

Reply:
I have updated the wiki somewhat, needs more work.